src/components/QueryBuilder.vue

```diff
@@ -119,6 +119,13 @@ export default Vue.extend( {
 				document
 					.getElementsByClassName( 'querybuilder__condition-wrapper-last' )[ 0 ]
 					.scrollIntoView( { behavior: 'smooth' } );
+
+				const toggle = document
+					.querySelectorAll( '.querybuilder__condition-relation-toggle .wikit-ToggleButton' );
+				const toggleElement = toggle[ toggle.length - 1 ] as HTMLElement;
+
+				toggleElement.setAttribute( 'tabindex', '0' );
```

Comment:
Could you explain why you are giving the button a tabindex?

Reply:
good catch. this is not needed at all, since the element is already focusable.